assets/admin/editor-wizard/style.scss

```diff
@@ -134,15 +134,16 @@
 	&__feature-item {
 		padding: 0 0 20px 24px;
 		margin: 0;
+		display: flex;
```

Comment:
This comes from a change I had from `feature/course-lesson-wizard` that @renatho's parent branch don't have yet. Please ignore all changes in this file.

Reply:
I must say that I was very confused for a moment because there are even commits from myself on this branch. But yeah, fair enough. :P